src/common/PageListPagination.tsx

```diff
@@ -0,0 +1,22 @@
+import { Pagination, Stack, StackProps } from '@mui/material'
+
+interface Props extends StackProps {
+  page: number
+  count: number
+  onChange: (page: number) => void
+}
+
+export default function PageListPagination({ page, count, onChange, ...rest }: Props) {
+  return (
+    <Stack alignItems='center' mt={6} {...rest}>
+      <Pagination
+        count={count}
+        page={page}
+        onChange={(_, page) => onChange(page)}
+        color={'primary'}
+        hidePrevButton
+        hideNextButton
+      />
+    </Stack>
+  )
+}
```

Comment:
changed my mind

let's not have it extracted to a separate component and use `Pagination` directly in the list (as we have here styles that shouldn't be common)

but let's specify `defaultProps` for `MuiPagination` component in `theme/components.ts` and set these values there:
```
color={'primary'}
hidePrevButton
hideNextButton
```